src/java/org/apache/cassandra/schema/MigrationManager.java

```diff
@@ -260,18 +260,7 @@ public static void resetLocalSchema()
 
         Schema.instance.clear();
 
-        Set<InetAddressAndPort> liveEndpoints = Gossiper.instance.getLiveMembers();
-        liveEndpoints.remove(FBUtilities.getBroadcastAddressAndPort());
-
-        // force migration if there are nodes around
-        for (InetAddressAndPort node : liveEndpoints)
-        {
-            EndpointState state = Gossiper.instance.getEndpointStateForEndpoint(node);
-            Future<Void> pull = MigrationCoordinator.instance.reportEndpointVersion(node, state);
-            if (pull != null)
-                FBUtilities.waitOnFuture(pull);
-        }
-
+        MigrationCoordinator.instance.reset();
```

Comment:
So reset in the coordinator was not used up to now... I am wondering why. It had also different meaning. Not sure whether it is ok to change the behavior of public method in a patch release, @adelapena, @bdeggleston, any thoughts?

Reply:
Actually the behavior is pretty similar